var/spack/repos/builtin/packages/py-deephyper/package.py

```diff
@@ -20,22 +20,28 @@ class PyDeephyper(PythonPackage):
     license("BSD-3-Clause")
 
     version("master", branch="master")
+    version("0.6.0", sha256="cda2dd7c74bdca4203d9cd637c4f441595f77bae6d77ef8e4a056b005357de34")
     version("0.4.2", sha256="ee1811a22b08eff3c9098f63fbbb37f7c8703e2f878f2bdf2ec35a978512867f")
 
     depends_on("python@3.7:3.9", type=("build", "run"))
```

Comment:
we may want to have `"python@3.7:3.11"` here, I am currently using `3.11` and it works fine.

Reply:
Should the Python version range be expanded just for v0.6.0, or also for v04.2?

Reply:
I have not tested 0.4.2 for a long time... and it can be considered unsupported now. You can adapt for `0.6.0` and later I guess.

Reply:
What's wrong with 3.12 and above? I don't see any such requirement in the package itself.